lib/parser.js

```diff
@@ -331,14 +337,13 @@ export default class Parser {
       nextToken = this.nextToken;
     }
 
-    let hasAt = indexesOf(word, '@');
-    let indices = sortAscending(uniq(flatten([[0], hasAt])));
-    let rNumber = /^((\d+(\.\d*)?)|(\.\d+))/;
+    hasAt = indexesOf(word, '@');
+    indices = sortAscending(uniq(flatten([[0], hasAt])));
 
     indices.forEach((ind, i) => {
-      let index = indices[i + 1] || word.length;
-      let value = word.slice(ind, index);
-      let node;
+      let index = indices[i + 1] || word.length,
+        value = word.slice(ind, index),
+        node;
 
       if (~hasAt.indexOf(ind)) {
```

Comment:
Could `hasAt.includes` be used instead?

http://node.green/#String-prototype-methods